test/unit/ready.js

```diff
@@ -1,73 +1,70 @@
-module("event");
+module( "event" );
 
-(function(){
+(function() {
```

Comment:
Seems to me that the best opportunity to change code styles is while we're already in that code, but I'm not overly adamant.